dev-util/gperf/files/gperf-3.1-clang-16-wregister.patch

```diff
@@ -0,0 +1,22 @@
+https://bugs.gentoo.org/882787
+
+a63b830554920476881837eeacd4a6b507632b19
```

Comment:
This is almost perfect (you even dropped the ChangeLog bit, which is good, because it makes patches hard to apply even if the ChangeLog bit applies _for now_), but in future, use the full git patch metadata.

https://git.savannah.gnu.org/gitweb/?p=gperf.git;a=commit;h=a63b830554920476881837eeacd4a6b507632b19 -> click "Patch" -> copy the full headers.

But I'll fix that on merge and it's really a nit. Also, include a link to the commit on some gitweb interface when possible.

Good job!